.github/workflows/ci.yml

```
@@ -42,7 +42,7 @@ jobs:
           args: >
             -kubernetes-version ${{ env.KUBERNETES_VERSION }}
             -schema-location default
-            -schema-location "./schemas/{{ .ResourceKind }}_{{ .ResourceAPIVersion }}.json"
+            -schema-location "https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json"
```

I think you can fix the line length lint like this:
```suggestion
            -schema-location
            "https://raw.githubusercontent.com/datreeio/CRDs-catalog/main/{{.Group}}/{{.ResourceKind}}_{{.ResourceAPIVersion}}.json"
```

---

Fixed by adding a `\` to break the string over into multiple lines.